translations/monero-core_pt-br.ts

```diff
@@ -1102,27 +1102,27 @@
     <message>
         <location filename="../pages/Settings.qml" line="525"/>
         <source>Wallet seed &amp; keys</source>
-        <translation type="unfinished"></translation>
+        <translation>Keys da semente</translation>
     </message>
     <message>
         <location filename="../pages/Settings.qml" line="527"/>
         <source>Secret view key</source>
-        <translation type="unfinished"></translation>
+        <translation>View key secreta</translation>
     </message>
     <message>
         <location filename="../pages/Settings.qml" line="528"/>
         <source>Public view key</source>
-        <translation type="unfinished"></translation>
+        <translation>View key pública</translation>
     </message>
     <message>
         <location filename="../pages/Settings.qml" line="529"/>
         <source>Secret spend key</source>
-        <translation type="unfinished"></translation>
+        <translation>Spend key secreta</translation>
```

Comment:
I tried to keep it consistent to the previous translation, so I kept spend, view and keys untranslated.
But I guess it could all be translated as:
keys =chaves
spend key = chave de gastos
view key = chave de visualização